Libraries/Image/RCTImageView.mm

```diff
@@ -420,9 +420,10 @@ - (void)reloadImage
     }
 
     RCTImageLoaderProgressBlock progressHandler = nil;
-    if (_onProgress) {
+    if (self.onProgress) {
+      RCTDirectEventBlock onProgress = self.onProgress;
       progressHandler = ^(int64_t loaded, int64_t total) {
-        self->_onProgress(@{
+        onProgress(@{
```

Comment:
no warnings based on this change?

Reply:
No warnings. This was intentional: https://github.com/facebook/react-native/commit/f12c8fba6059389e7e3c4bbce8cabd55d038e488